### PR TITLE
Add Yii Framework

### PR DESCRIPTION
**Pull request by jzfgo**
Hi, I've added Yii Framework. The logo is the one from [this page](https://github.com/topics/yii).

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -433,5 +433,12 @@
     "description": "xUnit.net is a free, open source, community-focused unit testing tool for the .NET Framework.",
     "image": "https://avatars2.githubusercontent.com/u/2092016?s=200&v=4",
     "language": "C#"
+  },
+  "Yii": {
+    "imports": ["yii\\", "yiisoft/yii"],
+    "technologies": ["Web Development", "PHP Frameworks"],
```
Comment (peti2001-test) [inline libraries.json]: Can you please also add "MVC"?

Comment (peti2001-test) [inline libraries.json]: technologies": ["Web Development", "PHP Frameworks"],

Comment (peti2001) [reply libraries.json]: Sorry, the code review was done by my test account. But anyway, Yii deserves an "MVC" tag too.

**Comment by jzfgo** [comment]
Sure. Here you go.

**Comment by peti2001** [comment]
Awesome, thanks for your contribution.

**Pull request merged**